js/index.js

```diff
@@ -106,3 +101,50 @@ messageForm.addEventListener('submit', function(event) {
      event.target.reset();
 
 });
+
+///lesson-15
```

Comment:
Good comment  - this is needed as it provides additional context about this pull request.